apps/kitty-krew/Dockerfile

```
@@ -2,35 +2,67 @@ FROM oven/bun:latest AS builder
 # Install dependencies
+RUN echo "Installing dependencies with bun"
 RUN bun install --frozen-lockfile
+RUN echo "Dependencies installed - node_modules contents:"
+RUN ls -la node_modules | head -n 10
+RUN echo "Total node_modules count: $(find node_modules -type f | wc -l)"
 
 # Copy application code
+RUN echo "Copying application code"
 COPY . .
+RUN echo "Application code copied - contents:"
+RUN ls -la
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)